07.SubpassBaking/main.cpp

```diff
@@ -237,17 +237,20 @@ class SubpassBaking : public ApplicationBase
 
             auto descriptorPool = logicalDevice->createDescriptorPoolForDSLayouts(video::IDescriptorPool::ECF_NONE, &gpuds1layout.get(), &gpuds1layout.get() + 1);
 
-            auto ubomemreq = logicalDevice->getDeviceLocalGPUMemoryReqs();
-            ubomemreq.vulkanReqs.size = neededDS1UBOsz;
 
             video::IGPUBuffer::SCreationParams cameraUBOCreationParams;
             cameraUBOCreationParams.usage = static_cast<asset::IBuffer::E_USAGE_FLAGS>(asset::IBuffer::EUF_UNIFORM_BUFFER_BIT | asset::IBuffer::EUF_TRANSFER_DST_BIT);
             cameraUBOCreationParams.canUpdateSubRange = true;
             cameraUBOCreationParams.sharingMode = asset::E_SHARING_MODE::ESM_EXCLUSIVE;
             cameraUBOCreationParams.queueFamilyIndexCount = 0u;
             cameraUBOCreationParams.queueFamilyIndices = nullptr;
+            cameraUBOCreationParams.size = neededDS1UBOsz;
+            cameraUBO = logicalDevice->createBuffer(cameraUBOCreationParams);
+
+            auto ubomemreq = cameraUBO->getMemoryReqs();
+            ubomemreq.memoryTypeBits &= logicalDevice->getPhysicalDevice()->getHostVisibleMemoryTypeBits();
```

Comment:
You can see on the previous API `ubomemreq = logicalDevice->getDeviceLocalGPUMemoryReqs();` was used,
Why did you use `getHostVisibleMemoryTypeBits` instead of `getDeviceLocalMemoryTypeBits`?